config.json

```diff
@@ -64,7 +64,9 @@
       "data": "cfg/coco.data",
       "cfg": "cfg/yolov4-tiny.cfg",
       "weights": "yolov4-tiny.weights"
-    }
+    },
+    "retry_delay_ms":30,
+    "max_retries_seconds":180
```

Comment:
Like it better this way. I wonder how much effort it will be to group all Darknet related configs here such as `PATH_TO_YOLO_DARKNET`, `CMD_TO_YOLO_DARKNET`, `darknet_json_stream` and `darknet_mjpeg_stream`

Reply:
Sorry, I just saw this.  I created a `MONITOR_SETTINGS` group as they didn't really fit anywhere else and they also get their own section in the docs.  As far as grouping darknet config, it shouldn't be too difficult.  I would need to find all the references and update them.  I figure that should be a separate PR for tracking.  I can work on that next week.